sa/database.go

```diff
@@ -34,9 +34,21 @@ var dialectMap = map[string]interface{}{
 func NewDbMap(driver string, dbConnect string) (*gorp.DbMap, error) {
 	logger := blog.GetAuditLogger()
 
-	// We require this parameter for MySQL, so fail now if it is not present
-	if driver == "mysql" && !strings.Contains(dbConnect, "parseTime=true") {
-		return nil, fmt.Errorf("Database connect string must have parseTime=true")
+	if driver == "mysql" {
+		// Check the parseTime=true DSN is present
+		dbURI, err := url.Parse(dbConnect)
+		if err != nil {
+			return nil, err
+		}
+		dsnVals, err := url.ParseQuery(dbURI.RawQuery)
+		if err != nil {
+			return nil, err
+		}
+		if k := dsnVals.Get("parseTime"); k != "true" {
+			dsnVals.Set("parseTime", "true")
+			dbURI.RawQuery = dsnVals.Encode()
+			dbConnect = dbURI.String()
```

Comment:
pull this out of the if-statement and pop it below. If it causes weird problems (because the parse/String cycle maybe changes things), it will do so consistently with or without parseTime being present.